build/shimplify.py

```diff
@@ -0,0 +1,343 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""implementation of shimplify Ant task in Python2.7 for Jython"""
+
+import errno
+import json
+import logging
+import os
+import re
+import subprocess
+
+# wrap Ant interactions to reduce linting warnings
+# TODO pass only Python types if possible
+
```

Comment:
Would be nice to put this comment in front of each of the related methods.  As it is, there was enough whitespace between this and the methods that initially I thought this comment was unrelated -- it has the same whitespace break as used between unrelated methods.

Maybe better would be adding this comment to the doc string of the methods so it's clearly tied to the method and obvious, even via generated docs, what's going on.